packages/transport-native-bluetooth/src/api/bluetoothManager.ts

```diff
@@ -46,6 +46,15 @@ const errorLog = (...args: any[]) => {
     console.error('BluetoothManager', ...args);
 };
 
+const toBluetoothDevice = (device: Device): BluetoothDevice => ({
+    id: device.id,
+    name: device.name ?? 'Unknown',
+    // Bluetooth company identifier is not expected to be present
```

Comment:
The comment is not clear. If I don't know that the first two bytes are Company Identifier, it's not obvious that it is related to the slicing. Could you please mention that in the comment, and ideally also mention why it is not expected? Is it just simplification, or size optimization, or just for consistency?

something like
```suggestion
    // First 2 bytes contain Company Identifier that is dropped because ... 
```

Reply:
Fixed.